docs/quality-assurance.md

```diff
@@ -0,0 +1,38 @@
+
+# Quality Assurance
+After migrating the collection you should verify if the process worked as expected. This guide assists you to verify if articles are not available at the new Website.
+
+## Verifying articles' availability at the new Website
+
+Use a list contain the articles identifiers (also known as PID) that you want to check. Also, you need to produce an URL that to be used to check if the articles are accessible. For example, you can define certain URL templates:
```

Comment:
Use a list **containing** article identifiers

Reply:
URL that **is** be used

Reply:
@robertatakenaka, o que você acha do trecho:

"Also, you need to produce an URL **to be used** to check if the articles are accessible. For example, you can define certain URL templates:"?

Reply:
sim, está correto.
No meu comentário, eu queria dizer "is to be", faltou o "to".

Reply:
Feito em 0c0ca865ce8d2d7fbf396945a776a11386ae20ba.